src/teleport.js

```diff
@@ -0,0 +1,37 @@
+import { checkForAlpine } from './utils'
+
+const AlpineTeleportMagicMethod = {
+    start() {
+        checkForAlpine()
+        Alpine.addMagicProperty('teleport', ($el) => {
+            return function (target = document.body, $element = $el, prepend = false) {
+                // find the portal
+                const destination = document.querySelector(`[x-portal="${target}"]`)
+
+                // throw error if portal is undefined
+                if (destination === undefined) {
```

Comment:
```suggestion
                if (!destination) {
```

Reply:
Well, there was a lot `try and see` went there :) thanks.